### PR TITLE
AB#12283 Accessibility Changes for Admin Page

### PR DESCRIPTION
**Pull request by pradnya13705**
Accessibility Changes for the Desktop application Admin page

### CODE REVIEW

File: file-share-service-desktop-client/Modules/Admin/AdminView.xaml [hunk 1]
```diff
@@ -16,13 +16,13 @@
                 <Setter Property="FontSize" Value="16" />
             </Style>
 
-            <Style TargetType="TextBlock" x:Key="KeyValuePairValueTextBlockStyle">
+            <Style TargetType="{x:Type TextBox}" x:Key="KeyValuePairValueTextBlockStyle">
                 <Setter Property="FontSize" Value="16" />
                 <Setter Property="Margin" Value="4 0" />
-                <Setter Property="TextTrimming" Value="CharacterEllipsis" />
+                <!--<Setter Property="TextTrimming" Value="CharacterEllipsis" />-->
```
Comment (jimjamjahaa) [inline file-share-service-desktop-client/Modules/Admin/AdminView.xaml:22]: should probably just be deleted

File: file-share-service-desktop-client/Modules/Admin/AdminView.xaml [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
                 <Setter Property="TextWrapping" Value="NoWrap" />
                 <Setter Property="FontWeight" Value="Bold" />
-                <Setter Property="ToolTip" Value="{Binding RelativeSource={RelativeSource Self}, Path=Text}" />
+                <!--<Setter Property="ToolTip" Value="{Binding RelativeSource={RelativeSource Self}, Path=Text}" />-->
```
Comment (jimjamjahaa) [inline file-share-service-desktop-client/Modules/Admin/AdminView.xaml:25]: should probably just be deleted

**Review by jimjamjahaa — COMMENTED** [review]
Couple of commented out lines in there which i don't like.

I notice there seems to be a lot of inline style code that gets repeated. I don't know much about xaml so... dealers choice as far as i'm concerned.

File: file-share-service-desktop-client/Modules/Admin/AdminView.xaml (hunk 1 above)
Comment (pradnya13705) [reply file-share-service-desktop-client/Modules/Admin/AdminView.xaml]: Commented code removed

File: file-share-service-desktop-client/Modules/Admin/AdminView.xaml (hunk 2 above)
Comment (pradnya13705) [reply file-share-service-desktop-client/Modules/Admin/AdminView.xaml]: Commented code removed

Comment (pradnya13705) [reply file-share-service-desktop-client/Modules/Admin/AdminView.xaml]: Where ever possible I have removed the inline style

**Review by jimjamjahaa — APPROVED** [review]
lgtm

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**